public/components/query_compare/search_result/search_components/search_configs/search_config.tsx

```diff
@@ -110,6 +150,23 @@ export const SearchConfig: FunctionComponent<SearchConfigProps> = ({
       </EuiTitle>
       <EuiSpacer size="m" />
       <EuiFlexGroup>
+        {dataSourceEnabled && (
+          <EuiFlexItem>
+            <EuiFormRow 
+              fullWidth
+              label="Data Source"
+            >
+            <DataSourceSelector
```

Comment:
Is there any plan to link the data source selector with the other two dropdowns (index and pipeline)? For example, if a datasource isn't selected, should we disable the other two since they would be empty?

Reply:
It is already linked together, when the datasource is changed the index and pipelines are fetched based on the new datasourceId

Reply:
> For example, if a datasource isn't selected, should we disable the other two since they would be empty?

Wanted to put more emphasis on this. If datasources are enabled, and a datasource hasn't been selected, should there be some type of guard that stops user from interacting with the index or pipeline dropdown, since they won't contain anything?

Reply:
if the data-source is not enabled it will revert to the old flow of being connected to the local cluster